source/09-audit/index.rst

```diff
@@ -7,15 +7,17 @@
 Audit
 =====
 
-An important feature of the COUNTER Code of Practice is that compliant content providers (including third-party services providing stats on behalf of content providers) MUST be independently audited on an annual basis in order to maintain their COUNTER-compliant status. To facilitate this, a set of auditing standards and procedures has been published in Appendix E of this Code of Practice. COUNTER has tried to meet the need of customers for credible usage statistics without placing an undue administrative or financial burden on content providers. For this reason, audits will be conducted online in accordance with the program included in the auditing standards and procedures (:ref:`Appendix E <appendix-e>`).
+R5.1 of the COUNTER Code of Practice, published in April 2023, will become the only valid version of the Code of Practice from 1 January 2025.
 
-The independent audit is REQUIRED within six months of a content provider’s first self-certifying their compliance with the COUNTER Code of Practice, and annually thereafter. COUNTER will recognize an audit carried out by any Certified Public Accountant (CPA) in the USA, by any Chartered Accountant (CA) in the UK, or by their equivalent in other countries. Alternatively, the audit may be done by a COUNTER-approved auditor which is not a CA or a CPA. (Contact COUNTER for a list of approved auditors.)
+An important feature of the COUNTER Code of Practice is that compliant report providers (including third-party services providing stats on behalf of publishers) MUST be independently audited on an annual basis to maintain their COUNTER-compliant status. To facilitate this, a set of auditing standards and procedures has been published in Appendix E of this Code of Practice. COUNTER has tried to meet the need of report consumers for credible usage statistics without placing an undue administrative or financial burden on report providers. For this reason, audits will be conducted online in accordance with the program included in the auditing standards and procedures (:numref:`e-audit-requirements-and-tests`).
```

Comment:
numref doesn't work for the appendices, instead :ref:`Appendix E <appendix-e>` must be used.